arrows/core/associate_detections_to_tracks_threshold.h

```diff
@@ -31,48 +31,44 @@ class KWIVER_ALGO_CORE_EXPORT associate_detections_to_tracks_threshold
   virtual ~associate_detections_to_tracks_threshold() noexcept;
 
   /// Get this algorithm's \link vital::config_block configuration block \endlink
-  /**
-   * \returns \c config_block containing the configuration for this algorithm
-   *          and any nested components.
-   */
+  ///
+  /// \returns \c config_block containing the configuration for this algorithm
+  ///          and any nested components.
   virtual vital::config_block_sptr get_configuration() const;
 
   /// Set this algorithm's properties via a config block
-  /**
-   * \throws no_such_configuration_value_exception
-   *    Thrown if an expected configuration value is not present.
-   * \throws algorithm_configuration_exception
-   *    Thrown when the algorithm is given an invalid \c config_block or is'
-   *    otherwise unable to configure itself.
-   *
-   * \param config  The \c config_block instance containing the configuration
-   *                parameters for this algorithm
-   */
+  ///
+  /// \throws no_such_configuration_value_exception
+  ///    Thrown if an expected configuration value is not present.
+  /// \throws algorithm_configuration_exception
+  ///    Thrown when the algorithm is given an invalid \c config_block or is'
```

Comment:
Weird spacing here?

Reply:
The spacing is the same as in the original.  This commit is only trying to change the comment characters used and leave as much unchanged as possible.

Reply:
Strictly speaking, there should be three spaces (one always following `//`, and two for indentation) here, not four. I'm okay leaving it as-is though if the extra space was also present in the original. (Yes, we allow the indent rules to get sloppy for some directives, effectively treating the exception type as part of the "function name", rather than as an argument. If, however, the description text started on the previous line, I would expect the continuation to be indented after `\throws`. Although I would probably discourage that in general in favor of this style.)